.github/workflows/lint.yaml

```diff
@@ -104,6 +104,7 @@ jobs:
           node-version: "22"
       - name: Run `make format-check`
         id: format-check
+        continue-on-error: true
```

Comment:
@ytimocin - there were some bugs in this check. See failed run here: https://github.com/radius-project/radius/actions/runs/12552709770/job/34999066289?pr=8177